tests/misc_env/install_prereq.py

```diff
@@ -117,6 +117,13 @@ def install_prereq(
     log.info(
         "distro version_id: {version_id}".format(version_id=distro_info["VERSION_ID"])
     )
+    # For baremetal deployents, change hostname to shortname
+    cmd_shortname = "sudo hostname `hostname -s`"
+    ceph.exec_command(cmd=cmd_shortname)
```

Comment:
Can we avoid this? Down the code workflow we have a system reboot to pick the new kernel changes and it will render this not effective.

`hostname` is non persistence change. Also, `cephadm` expects us to match with the name with the output of `hostname`. There is a larger PR that is expected to correct the behavior in the code.

Hence, can we avoid this change now?

Reply:
Sure. We can take this down for now.